awd/ssm.py

```diff
@@ -0,0 +1,17 @@
+from boto3 import client
+
+
+class SSM:
```

Comment:
For readability (especially down the road or in case somebody else is maintaining this), it may be helpful to either change this to a more descriptive (but succinct) name or add a comment for what SSM stands for. (Same for the operation on config.py line 17.)

Reply:
Updated the docstring with more detail about SSM, let me know if that provides enough clarity